### PR TITLE
Add Asus ROG G75VX.xml

### PR DESCRIPTION
**Pull request by RypeDub420**
Asus ROG G75VX Temperature Steps & Fan Speeds by: RypeDub - https://www.YouTube.com

### CODE REVIEW

**Comment by RypeDub420** [comment]
Before you implement this into the release:

Does each EC (motherboard) have a specific set of values that will control the fan, or is this program specifically written to add more steps to the fan speeds, based on user definable temperatures?

For instance: let's say the "default" way my fan spins, only has 5 steps of temperatures and 5 fan speed values. Can I use this program to add 5 or even 10 more temperature steps and add 5 or 10 more fan speeds to more evenly spin the fans down or up?

Or is that like a fan hardware limitation: they only have 5 steps / values?

The only reason I am asking this is becasue from what I know about Electronics: as long as you can control the voltage: you can control the fan speed.

So it looks like this program is what you would use to add more to whatever your manufacture defined.

Am I correct?

If yes: let me add a total of 15 steps (the "smallest" divisor near 10 of 255) so that us overclockers (who bought this WITHOUT overclocking it lol) will have less noise and better speeds at more practical temperatures.

I've been overclocking this thing for over 4 years and I know which temperatures the fans should be spinning, and at what speeds these fans should be spinning at.

HOWEVER: I am ONLY going to write my configuration geared towards the OverClocker, so the steps will be very "generic" in the fashion that every 5 degrees causes another step up or down, and a total of 7 degrees will need to be changed in order for the fans to actually spin up or down.

This is the MOST practical for OverClockers who want to use an extreme overclock and need the fans at 100% near 80% (I will adjust for the 15 increments) or for silent UnderClockers who don't wanna hear the fan, thus having a much smaller fan spin speed of 16.6% (which is the default lowest).

**Comment by RypeDub420** [comment]
@hirschmann: could you add the config label please?

**Comment by hirschmann** [comment]
You are correct, NBFC overrides the EC's default behavior. You can add as many thresholds as you like.

However, the config file you've committed is invalid (it's not even a valid XML file). It looks like a failed attempt to merge 2 files.

**Comment by RypeDub420** [comment]
LOL It's definitely an XML file, all I did was fork your repo, then upload my XML conf, then I made an update to it (which is the one I'd like live) by just editing the file. Travis CI and AppVeyor said it passed.

How do I delete things from my branch, so that only my XML is there? Is that necessary?

File: Configs/Asus ROG G75VX.xml [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+          <FanSpeedValue>255</FanSpeedValue>
+          <TargetOperation>ReadWrite</TargetOperation>
+        </FanSpeedPercentageOverride>
+      </FanSpeedPercentageOverrides>
+    </FanConfiguration>
+  </FanConfigurations>
+  <RegisterWriteConfigurations />
+</FanControlConfigV2>
+
+        <TemperatureThreshold>
```
Comment (hirschmann) [inline Configs/Asus ROG G75VX.xml]: Lines 155-235 are redundant.

Comment (RypeDub420) [inline Configs/Asus ROG G75VX.xml]: All I was originally to make the XML was literally copy one of the other ROG configs with the same registers 150 and 151 for the CPU/GPU EC's.

So maybe some of the other files are also redundant?

**Comment by RypeDub420** [comment]
LOL Must have not removed the previous lines before pasting. This time it's 153 lines.

**Comment by hirschmann** [comment]
Looks good now. Thanks for your effort 👍

**Comment by RypeDub420** [comment]
YEEE!

**Pull request merged**